src/content/changes/highlights/2024-07-21.md

```diff
@@ -0,0 +1,186 @@
+        - Update Default Zot container image tag to version [v2.1.0](https://github.com/project-zot/zot/releases/tag/v2.1.0)
+
+- [aws-cloud-controller-manager-app](https://github.com/giantswarm/aws-cloud-controller-manager-app)
+    - [1.29.3-gs1](https://github.com/giantswarm/aws-cloud-controller-manager-
+app/compare/v1.28.6-gs1...v1.29.3-gs1)
+        - Chart: Update to upstream v1.29.3. ([#62](https://github.com/giantswarm/aws-cloud-controller-manager-app/pull/62))
+- [cluster-api-provider-azure-app](https://github.com/giantswarm/cluster-api-provider-azure-app)
+    - [1.12.4-gs2](https://github.com/giantswarm/cluster-api-provider-azure-
+app/compare/v1.12.4-gs1...v1.12.4-gs2)
+        - Remove kube-rbac-proxy from azure-service-operator.
+        - Use image of azure-service-operator from `gsoci.azurecr.io`.
+- [cluster-azure](https://github.com/giantswarm/cluster-azure)
+    - [0.16.1](https://github.com/giantswarm/cluster-
+azure/compare/v0.16.0...v0.16.1)
+        - Respect `global.apps.externalDnsPrivate` to overwrite configuration of `external-dns-private` app.
+- [cloud-provider-cloud-director-app](https://github.com/giantswarm/cloud-provider-cloud-director-app)
+    - [0.3.0](https://github.com/giantswarm/cloud-provider-cloud-director-
+app/compare/v0.2.11...v0.3.0)
+        - Rollback CPI from `1.6.0` to `1.5.0` due to IP Spaces incompatibility.
+- [cluster-cloud-director](https://github.com/giantswarm/cluster-cloud-director)
+    - [0.56.1](https://github.com/giantswarm/cluster-cloud-
+director/compare/v0.56.0...v0.56.1)
+        - Rollback CPI from `1.6.0` to `1.5.0` due to VCD version incompatibility.
+    - [0.56.0](https://github.com/giantswarm/cluster-cloud-
+director/compare/v0.55.0...v0.56.0)
+        - Bump Kubernetes to `1.27.14`.
+    - [0.55.0](https://github.com/giantswarm/cluster-cloud-
+director/compare/v0.54.1...v0.55.0)
+        - Bump Kubernetes to `1.26.14`.
+- [cluster-vsphere](https://github.com/giantswarm/cluster-vsphere)
+    - [0.56.1](https://github.com/giantswarm/cluster-
+vsphere/compare/v0.56.0...v0.56.1)
+        - Unpause Cluster resource as part of cleanup hook after deletion in order to prevent leftover resources.
+- [default-apps-cloud-director](https://github.com/giantswarm/default-apps-cloud-director)
+    - [0.9.0](https://github.com/giantswarm/default-apps-cloud-
+director/compare/v0.8.0...v0.9.0)
+        - Update `cert-exporter` to v2.9.1.
+        - Update `cert-manager-app` to v3.8.0.
+        - Update `k8s-dns-node-cache-app` to v2.8.1.
+        - Update `net-exporter to v1.21.0`
+        - Update `observability-bundle` to v1.4.0.
+        - Update `security-bundle` to v1.7.1.
+        - Update `teleport-kube-agent-app` to v0.9.2.
+        - Update `vertical-pod-autoscaler-app` to v5.2.4.
+- [azure-private-endpoint-operator](https://github.com/giantswarm/azure-private-endpoint-operator)
+    - [0.2.3](https://github.com/giantswarm/azure-private-endpoint-
+operator/compare/v0.2.2...v0.2.3)
```

Comment:
@T-Kukawka These are from Phoenix. Please adapt the content to the standard format (name, version, description)
[example](https://raw.githubusercontent.com/giantswarm/docs/main/src/content/changes/highlights/2024-02-15.md)

Reply:
Done